content/docs/command-reference/config.md

```diff
@@ -179,11 +179,11 @@ This section contains the following options, which affect the project's
   > set.
 
 - `cache.shared` - permissions for newly created or downloaded cache files and
-  directories. The default is `0o664`(rw-r--r--) for files and `0o755`
-  (rwxr-xr-x) for directories. The only accepted value right now is `group`,
-  which makes DVC use `0o664` (rw-rw-r--) for files and `0o775` (rwxrwxr-x) for
-  directories, which is useful when you are using a a
-  [shared development server](/doc/use-cases/shared-development-server).
+  directories. The only accepted value right now is `group`, which makes DVC use
+  `0o664` (rw-rw-r--) for files and `0o775` (rwxrwxr-x) for directories, which
```

Comment:
Do we need the `0o` octal notation? Most people who know about UNIX file permissions will be familiar with plain `664`/`775` probably?

Reply:
Also I wonder what this does in Windows... @efiop do you know? Thanks

Reply:
I also don't think we need `0o` at the beginning.

Reply:
Don't know about windows, python only operates with user read-only bit, so unix perms are almost meaningless. As well as shared development server scenario is probably meaningless, it was never tested on windows. Let's just keep the *nix perms for now.